types/jest/index.d.ts

```diff
@@ -1082,10 +1080,7 @@ declare namespace jest {
     ExpectProperties &
     AndNot<CustomAsyncMatchers<TMatchers>> &
     ExtendedExpectFunction<TMatchers>;
-    /**
-     * Construct a type with the properties of T except for those in type K.
-     */
-    type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
+
```

Comment:
This change should only be in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49969>.

Reply:
Huh that's weird - I made this via the GH UI, so not sure why it decided to do the edit off my branch for #49969 instead of master.

I'll rebase :)